### PR TITLE
Add CUB support in Jenkins

### PR DESCRIPTION
**Pull request by emcastillo**

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

File: run_test.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def main():
         'chainer-py2', 'chainer-py3', 'chainer-py35', 'chainer-slow',
         'chainer-example', 'chainer-prev_example', 'chainer-doc',
         'chainer-head',
-        'cupy-py2', 'cupy-py3', 'cupy-py35', 'cupy-slow',
+        'cupy-py2', 'cupy-py3', 'cupy-py35', 'cupy-slow', 'cupy-py37-cub',
```
Comment (niboshi) [inline run_test.py]: py38?

File: run_test.py [hunk 2]
```diff
@@ -376,12 +390,14 @@ def main():
         raise
 
     use_ideep = any(['ideep4py' in req for req in conf['requires']])
+    use_cub = 'cub' in args.test
```
Comment (niboshi) [inline run_test.py]: Instead, assign `use_cub = True` in the `elif` clause above.

File: run_test.py (hunk 1 above)
Comment (emcastillo) [reply run_test.py]: maybe py3-cub is better

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
@emcastillo Could you resolve conflict?

**Comment by emcastillo** [comment]
rebased!

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by emcastillo** [comment]
I think i deleted the use_cub declaration on the rebase by accident.

**Comment by emcastillo** [comment]
Fixed

**Pull request merged**